src/uiSelectController.js

```diff
@@ -257,8 +254,6 @@ uis.controller('uiSelectCtrl',
       if (ctrl.dropdownPosition === 'auto' || ctrl.dropdownPosition === 'up'){
         $scope.calculateDropdownPos();
       }
-
-      $scope.$broadcast('uis:refresh');
```

Comment:
Yes you could both  I cannot think of any use case where you need this? And if we want to support this it should be in the `refresh `function instead of `activate`.